src/autoload.php

```diff
@@ -10,66 +10,17 @@
 /**
  * Generate an autoloader for the WordPress file naming conventions.
  *
+ * @deprecated Use https://github.com/alleyinteractive/wordpress-autoloader or
```

Comment:
Would we want to add a `trigger_error` as well? From some light googling, that seems to be good thing to include. Not sure if that would affect any production projects.

Reply:
Great idea!